README.md

```diff
@@ -9,7 +9,7 @@ on.
 ## Pre-requisites
 
 - An Azure Arc-enabled CNCF K8s cluster.
-- Azure CLI `2.42.0` or higher. If you don't have az cli installed, follow [these instructions](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli).
+- Azure CLI `2.46.0` or higher. If you don't have az cli installed, follow [these instructions](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli).
```

Comment:
is there a reason for this?

Reply:
@vilit1 this is our [current min version](https://github.com/Azure/azure-iot-ops-cli-extension/blob/ffe03041b21b51b2ab1be5b8db287a31a3e65fb3/azext_edge/azext_metadata.json#L2)

Reply:
as in when did we bump the min version

Reply:
To add-on, we need at least 2.46 to ensure track 2 authorization client from CLI core.